packages/netsuite-adapter/src/config.ts

```diff
@@ -383,27 +393,45 @@ const additionalDependenciesConfigPath: string[] = [
 ]
 
 const validateAdditionalSdfDeployDependencies = (
-  { features, objects }: Record<keyof AdditionalSdfDeployDependencies, unknown>,
+  input: Partial<Record<keyof AdditionalSdfDeployDependencies, unknown>>,
   configName: string
-): void => {
+): input is Partial<AdditionalSdfDeployDependencies> => {
+  const { features, objects } = input
   if (features !== undefined) {
     validateArrayOfStrings(features, additionalDependenciesConfigPath.concat(configName, 'features'))
   }
   if (objects !== undefined) {
     validateArrayOfStrings(objects, additionalDependenciesConfigPath.concat(configName, 'objects'))
   }
+  return true
 }
 
 const validateAdditionalDependencies = (
-  { include, exclude }: Record<keyof AdditionalDependencies, unknown>
+  { include, exclude }: Partial<Record<keyof AdditionalDependencies, unknown>>
 ): void => {
+  let validatedInclude: Partial<AdditionalSdfDeployDependencies> = {}
   if (include !== undefined) {
     validatePlainObject(include, additionalDependenciesConfigPath.concat('include'))
     validateAdditionalSdfDeployDependencies(include, 'include')
+    validatedInclude = include
   }
```

Comment:
nit: you can extract this code to a typeguard to remove the need of the `validatedInclude`.
```suggestion
  isPartialAdditionalSdfDeployDependencies(include, 'include')
```